homeassistant/components/device_tracker/unifi.py

```diff
@@ -33,55 +40,87 @@
     vol.Required(CONF_USERNAME): cv.string,
     vol.Required(CONF_PORT, default=DEFAULT_PORT): cv.port,
     vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean,
+    vol.Optional(CONF_DETECTION_TIME, default=DEFAULT_DETECTION_TIME): vol.All(
+        cv.time_period, cv.positive_timedelta),
+    vol.Optional(CONF_API_VERSION, default=DEFAULT_API_VERSION): cv.string
```

Comment:
If you only allow two API versions, please update the config validator to only match that: `vol.In([API_V4, API_V5])`

Reply:
Agreed, I'm actually not super clear why v4 is being used by default either, v5 is current, and I would think we'd want to be using that.